extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,237 +25,272 @@ limitations under the License.
+Documents append to the end of the current document as a child of the `<amp-next-page>` element. To prevent shifting page content down, this component must be the last child of the document `<body>`. If needed, any footer content should be embedded inside the `<amp-next-page>` tag and will be displayed once no more article suggestions are available.
 
-| Event                  | Fired when                                               |
-| ---------------------- | -------------------------------------------------------- |
-| `amp-next-page-scroll` | The user scrolls to a new page                           |
-| `amp-next-page-click`  | The user click on an article from the recommendation box |
+#### Inline Configuration
 
-Both of the `triggers` provide the variables `fromUrl` and `toUrl` referring to the previous and current pages. They can be used as follows:
+The inline configuration defines the documents recommended by `amp-next-page` to
+the user as an ordered array of page objects in JSON format.
 
-```html
-<amp-analytics>
-  <script type="application/json">
+Each page object should have the following key/value pairs:
+
+| Key                | Value                                                                                                                                                            |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `url` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the Google AMP cache when required. |
+| `title` (required) | String title of the page, will be used when rendering the recommendation box.                                                                                    |
+| `image` (required) | String URL of the image to display in the recommendation box.                                                                                                    |
+
+To enable for an infinite loading experience without the need for a server-side configuration (remote-loading), `amp-next-page` automatically enables `deep-parsing` (see Attributes), which allows it to parse more suggestions by recursively looking at inline configurations inside `<amp-next-page>` tags on the loaded documents. To disable this behavior, set `deep-parsing` to `false`.
+
+The following configuration will recommend two more documents for the user to read.
+
+```json
+[
+  {
+    "image": "https://example.com/image1.jpg",
+    "title": "This article shows first",
+    "url": "https://example.com/article1.amp.html"
+  },
+  {
+    "image": "https://example.com/image2.jpg",
+    "title": "This article shows second",
+    "url": "https://example.com/article2.amp.html"
+  }
+]
+```
+
+#### Load configuration from a remote URL
```

Comment:
This content is structured a bit backwards. Start with the on-page configuration:

```
Use the src attribute to point to the remote JSON configuration.

<amp-next-page src="https://example.com/next-page-config.json"></amp-next-page>
```